pages/linux/sacct.md

```diff
@@ -18,3 +18,11 @@
 - Display elapsed time, job name, number of requested CPUs, and memory requested of a job:
 
 `sacct --jobs {{job_id}} --format={{elapsed}},{{jobname}},{{reqcpus}},{{reqmem}}`
+
+- Display recent jobs that occurred from one week ago up to the present day:
+
+`sacct --starttime=$(date -d "1 week ago" +'%F')`
+
+- Output a larger number of characters for an attribute:
+
+`sacct --format=JobID,JobName%100`
```

Comment:
```suggestion
`sacct --format={{job_id}},{{job_name}}%100`
```
I guess we can follow the same placeholder convention as the other examples.

Reply:
These are indeed not placeholders. `JobID` and `JobName` are names of columns output by sacct.

Reply:
> These are indeed not placeholders. `JobID` and `JobName` are names of columns output by sacct.

Thanks for the clarification.